src/Codeception/Test/Loader/Gherkin.php

```diff
@@ -58,7 +45,8 @@ public function __construct($settings = [])
         if (!class_exists('Behat\Gherkin\Keywords\ArrayKeywords')) {
             throw new TestParseException('Feature file can only be parsed with Behat\Gherkin library. Please install `behat/gherkin` with Composer');
         }
-        $keywords = new GherkinKeywords(['en' => static::$defaultKeywords]);
+        $i18n = require __DIR__ . '/../../../../../../behat/gherkin/i18n.php';
```

Comment:
this **hardcoded** path is really really bad. For instance it will work ONLY if behat and codeception both are in `vendor` dir of Codeception, but in Codeception repo, Codeception itself is in `src` fir, while behat is in `vendor/behat`.

Here you should get a path to Composer vendor dir and provide path from it